ha/remote_execution/ssh_communicator.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, hostname, username=None, port=22, password=None):
             Log.error(f'SSHRemoteExecutor, some error occured while connecting \
                         to SSH channel {err}')
 
-    def execute(self, command: str) -> None:
+    def execute(self, command: str, secret=None) -> None:
```

Comment:
Please check RemoteExecutor parent class of current class and update parent class as 
overridden method should have same definition

Reply:
RemoteExecutor's execute method updated.